src/logic.ts

```diff
@@ -57,15 +59,19 @@ export function success(data: fetch.Response, res: ExpressModule.Response) {
   if (Buffer.isBuffer(body)) {
     res.send(body)
   } else if (isBlob(body)) {
-    const blob = body as Blob
-    const stream = blob.stream()
-    if (stream.pipeTo) {
-      stream.pipeTo(res as any)
-    } else {
-      ;(blob.stream() as any).pipe(res)
-    }
+    // const blob = body as Blob
```

Comment:
why is this commented out?

Reply:
Because there is no test and I couldn't find a good use case for blobs in the server, I think it was meant for the browsers only. In any case, I prefered to fail explicitly